tests/Fixtures/project/src/Aspect/InitializationAspect.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Go\Tests\TestProject\Aspect;
+
+use Go\Aop\Aspect;
+use Go\Lang\Annotation as Pointcut;
+
+class InitializationAspect implements Aspect
+{
+    /**
+     * Before class instance initialization.
+     *
+     * @Pointcut\Before("initialization(Go\Tests\TestProject\Application\Main)")
```

Comment:
I do not know about that, however, I know that if we have it in code, we should have way to test it. Right?